src/tests/unit/onbase-serializer.test.js

```diff
@@ -3,9 +3,9 @@ import chaiAsPromised from 'chai-as-promised';
 import chaiSubset from 'chai-subset';
 import _ from 'lodash';
 
-const { openapi } = require('utils/load-openapi');
-const onBaseSerializer = require('api/v1/serializers/onbase-serializer');
-const testData = require('./test-data');
+import { openapi } from 'utils/load-openapi';
+import * as onBaseSerializer from 'api/v1/serializers/onbase-serializer';
```

Comment:
I think this should be imported before `import { openapi } from 'utils/load-openapi'`.

Reply:
Why is that? I thought it was alphabetical order based on the file name.

Reply:
@Ruefa oh ah, hmm, you are right, but I think the file name here also include the prefix path.

Reply:
For example: https://github.com/osu-mist/express-api-skeleton/blob/master/src/app.js#L13-L20